app/src/main/java/com/droidhats/campuscompass/roomdb/AppDB.kt

```diff
@@ -0,0 +1,30 @@
+package com.droidhats.campuscompass.roomdb
+
+import android.content.Context
+import androidx.room.Database
+import androidx.room.Room
+import androidx.room.RoomDatabase
+import androidx.sqlite.db.SupportSQLiteDatabase
+
+@Database(entities = [ShuttleBus_Loyola_Entity::class,ShuttleBus_SGW_Entity::class],version = 3, exportSchema = false)
+abstract class AppDB : RoomDatabase() {
```

Comment:
Would be preferable to change the name of this file as we will have more than 1 database schema

Reply:
Name updated to ShuttleBusDB